templates/homebrew.mxcl.buildkite-agent.plist.j2

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>Label</key>
```

Comment:
Would you mind laying out the plist so that the dictionary keys are alpha-sorted?

Doing that means it's easier to scan by eye to find the one being looked for, and it also helps to avoid merge-conflicts that otherwise are common if two people add an entry to the end of the list.